sdcm/cluster.py

```diff
@@ -766,6 +766,10 @@ def grafana_address(self):
         """
         return self.external_address
 
+    @property
+    def scylla_address(self):
```

Comment:
why this new addition ? please add a docstring for what it should be used

Reply:
Probably, I chose bad name for this property. It's added because bound address inside Scylla pod is not the same as used for connectivity from other pods.

Maybe `scylla_bound_address` will be better? What do you think?

Reply:
so why do we need a new name or alias for it ? this is basically what `ip_address` means and then we have the `external_address`

Reply:
Because for k8s we need 3 addresses: 1 internal pod IP (used for communication inside one pod, including localhost connection), 1 internal cluster IP (used for communication between pods) and 1 external IP.

The problem that we can't use 127.0.0.1 for local cqlsh connections, because Scylla not bound to localhost.

Reply:
O.k.but would love to see those notes in that function, so it would be clear when it should be used. I.e. mostly for nodetool (and I guess also for API calls directly to scylla, internally from the node, like i.e. call to port 10000, like aborting repairs)

Reply:
Sure.